climpred/metrics.py

```diff
@@ -240,38 +287,260 @@ def _pearson_r_p_value(forecast, reference, dim=None, **metric_kwargs):
     positive=False,
     probabilistic=False,
     unit_power=0.0,
-    long_name="Pearson's Anomaly correlation coefficient p-value",
-    aliases=['p_pval', 'pvalue', 'pacc'],
+    long_name='Pearson product-moment correlation coefficient p value',
+    aliases=['p_pval', 'pvalue', 'pval'],
     minimum=0.0,
     maximum=1.0,
     perfect=0.0,
 )
 
 
-def _spearman_r(forecast, reference, dim=None, **metric_kwargs):
+def _effective_sample_size(forecast, reference, dim=None, **metric_kwargs):
```

Comment:
does this go eventually into `xskillscore`?

Reply:
I know you need it fast in here, but in a month it can be transfered, can't it?

Reply:
Yes this could all go over to `xskillscore`. I just figured it was a bit specialized to be included over there and might as well be over here.

Reply:
I wouldn’t say it’s too specialised for xs